src/routes/home/index.tsx

```diff
@@ -1,8 +1,18 @@
 import { FunctionalComponent, h } from "preact";
 import style from "./style.css";
 import SelectElectorate from "../../components/select-electorate";
+import { useEffect } from "preact/hooks";
+import { route } from "preact-router";
 
 const Home: FunctionalComponent = () => {
+
+  useEffect(() => {
+    let selectedElectorate = localStorage.getItem('electorate')
+    if(selectedElectorate){
```

Comment:
Can you run prettier on the files you changed (`npx prettier --write .`)? It should fix the formatting, e.g. here and the line above (missing semicolon).

Reply:
This is done in the new commit